Friendly property names (#949)
osisoft/OCS-Docs: content/overview/get-started/view-profile.md

## Show user-friendly property names

> The **Show user friendly property names** option controls how property names for streams that originate through PI to OCS agents appear on the Trend and Asset Explorer pages. When this option is turned on, those stream property names mimic PI tag naming and certain stream properties are hidden. For example, say you have the following streams and properties:

Reviewer: ```suggestion
The **Show user friendly property names** option controls the appearance of property names for streams that originate through PI to OCS agents on the `Trend` and `Asset Explorer` pages. When this option is turned on, those stream property names mimic PI tag naming and certain stream properties are hidden. For example, say you have the following streams and properties:
```

Reviewer: The original is okay, but I found the big separation between "how" and "appear" awkward.

 - SL-Tank01|Pressure|Value

 - SL-Tank01|Pressure|SystemStateCode

 - SL-Tank01|Temperature|Value
 
 - SL-Tank01|Temperature|SystemStateCode

> With this option turned on, you would see the following on the Trend and Asset Explorer pages:

Reviewer: ```suggestion
With this option turned on, you would see the following on the `Trend` and `Asset Explorer` pages:
```
